# Patch made by Salonia Matteo <saloniamatteo@protonmail.com>
# Made for saluto-lightdm-theme-dwm
# Apply to saluto/src/components/SelectItem.vue
--- SelectItem.vue	2020-11-22 09:29:06.970922306 +0100
+++ SelectItem.vue	2020-11-22 09:30:30.937969341 +0100
@@ -72,6 +72,8 @@
                     icon = "liri";
                 } else if (key.indexOf("i3") > -1) {
                     icon = "i3";
+                } else if (key.indexOf("dwm") > -1) {
+                    icon = "dwm";
                 } else if (key.indexOf("xmonad") > -1) {
                     icon = "xmonad";
                 } else if (key.indexOf("budgie") > -1 || key.indexOf("solus") > -1) {
